packages/celotool/src/cmds/geth/simulate_client.ts

```diff
@@ -65,7 +65,7 @@ export const builder = () => {
     .options('test-mode', {
       type: 'string',
       description: 'Load test mode: mixed transaction types, big calldatas, or simple transfers',
-      choices: [TestMode.Mixed, TestMode.Data, TestMode.Transfer],
+      choices: [TestMode.Mixed, TestMode.Data, TestMode.Transfer, TestMode.StableTransfer],
```

Comment:
In the description you can add that "stable_transfer" is for stable transfers paid in stables

Reply:
Sure, updated the description.